src/sys/uio.rs

```diff
@@ -56,6 +59,38 @@ pub fn pread(fd: RawFd, buf: &mut [u8], offset: off_t) -> Result<usize>{
     Errno::result(res).map(|r| r as usize)
 }
 
+// process_vm_{read,write}v currently doesn't use flags, but we define this
```

Comment:
Let's remove flags. We can assess a breaking change or adding `*_flags` when/if this moment comes to pass in the future.